QUEST.tsv

```diff
@@ -1431,8 +1431,8 @@ QUEST_20150323_001431	It's not the right amount.
 QUEST_20150323_001432	Worship the Goddess Statue to receive blessings or move on to another area.
 QUEST_20150323_001433	What a nasty curse. {nl}Let's see which is stronger. The curse of Naktis or the purifier of the Goddess.
 QUEST_20150323_001434	Of course I have the purifier. {nl}But I'm not giving it to you unless you offer me money.
-QUEST_20150323_001435	{memo X}It could be a broken plaster of the Goddess Statue so get rid of the monsters there. {nl}If I'm right, then Naktis must be behind it. {nl}I may be far away but I hear the stories.
-QUEST_20150323_001436	{memo X}Naktis is looking for a clue to enter the Cathedral. {nl}If I'm right, not only has the Forest of Prayer been affected, but all of Pilgrim Path. {nl}If you reach the Cathedral, please defeat Naktis. {nl}You have to stop the revelation from falling into demon hands.
+QUEST_20150323_001435	{memo X}They could steal the gypsum Goddess Statue, so get rid of the monsters there. {nl}If I'm right, then Naktis must be behind it. {nl}I may be far away but I can still hear of them.
```

Comment:
The translation is off here. Literal version is "It could have been the one who stole the gypsum statue of the Goddess, so get rid of the monsters in that place. If my prediction is right, then Naktis is behind it. It might be far away, but I just heard that." This is based on KR->JP.